testing/cockroach/APKBUILD

```diff
@@ -0,0 +1,95 @@
+# Contributor: Nathan Johnson <nathan@nathanjohnson.info>
+# Maintainer: Nathan Johnson <nathan@nathanjohnson.info>
+pkgname=cockroach
+pkgver=0.1_beta20161201
+_pkgrealver=beta-${pkgver##*_beta}
+pkgrel=0
+pkgdesc="Distributed SQL database"
+url="https://www.cockroachlabs.com"
+arch="all"
+license="ASL 2.0"
+depends=""
+depends_dev=""
+makedepends="$depends_dev go go-tools autoconf bash linux-headers libseccomp libseccomp-dev glide"
+install="$pkgname.pre-install"
+subpackages="$pkgname-doc"
+pkgusers="cockroach"
+pkgroups="cockroach"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/cockroachdb/cockroach/archive/$_pkgrealver.tar.gz
+	$pkgname.initd
+	$pkgname.confd
+	$pkgname.logrotate
+	setversion.patch
+	10130_musl_strptime.patch
+	disable_style_test.patch
+	relax_timeout.patch
+	ldflags_fix.patch"
+builddir="$srcdir/go/src/github.com/cockroachdb/cockroach"
+options="!strip"
+
+prepare() {
+	mkdir -p $(dirname $builddir)
+	mkdir "$srcdir/go/pkg"
+	mkdir "$srcdir/go/bin"
+	mkdir "$srcdir/go/localpkg"
+	mv "$srcdir/$pkgname-$_pkgrealver" "$builddir"
+	default_prepare
+}
+
+build() {
+	export GOPATH="$srcdir/go"
+	cd $builddir
+	glide install
+	# force an auto reconf, since it seems this ships with some
+	# artificats from an autoconf run for a glibc distro
```

Comment:
Yeah, this is not correct. For one, we already ship various artifacts for various platform; it would be trivial to add one for alpine/musl.

That said, I've built this against musl myself, so I don't know why this is required.